test/run.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
-set -e
+set -euo pipefail
 
 CWD=$(pwd)
 CMD="elm-review --no-color"
 TMP="$CWD/temporary"
 ELM_HOME="$TMP/elm-home"
 SNAPSHOTS="$CWD/run-snapshots"
-SUBCOMMAND="$1"
+SUBCOMMAND="${1:-}"
```

Comment:
This makes the subcommand default to none if there is none passed, for `nounset`.